Formula/glib.rb

```diff
@@ -62,6 +63,21 @@ def install
               "giomoduledir=#{HOMEBREW_PREFIX}/lib/gio/modules",
               "giomoduledir=${libdir}/gio/modules"
 
+    # Delete python files because they are provided by `glib-utils`
+    dummy_python_shebang_rewrite_info = python_shebang_rewrite_info("#{HOMEBREW_PREFIX}/bin/python3")
+    prefix.glob("**/*").each do |f|
```

Comment:
I think this is equivalent.
```suggestion
    prefix.find do |f|
```